MAUI/MauiAppB2C/MSALClient/PCAWrapperB2C.cs

```diff
@@ -21,12 +28,20 @@ private PCAWrapperB2C()
             // Create PCA once. Make sure that all the config parameters below are passed
             PCA = PublicClientApplicationBuilder
                                         .Create(B2CConstants.ClientID)
+                                        .WithLogging(LogHere)
                                         .WithB2CAuthority(B2CConstants.AuthoritySignInSignUp)
                                         .WithIosKeychainSecurityGroup(B2CConstants.IOSKeyChainGroup)
-                                        .WithRedirectUri(PlatformConfig.Instance.RedirectUri)
+                                        .WithRedirectUri(B2CConstants.RedirectUri)
                                         .Build();
         }
 
+        // This demos logging
+        private void LogHere(LogLevel level, string message, bool containsPii)

```

Comment:
Let's no longer showcase this logger. Instead, use the newer logger interface that Travis added https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/logging